polars/polars-core/src/chunked_array/ops/unique/rank.rs

```diff
@@ -298,21 +312,21 @@ mod test {
             &[Some(1), Some(2), Some(3), Some(2), None, None, Some(0)],
         );
 
-        let out = rank(&s, RankMethod::Average)
+        let out = rank(&s, RankMethod::Average, false)
             .f32()?
             .into_iter()
             .collect::<Vec<_>>();
 
         assert_eq!(
             out,
             &[
-                Some(2.0f32),
```

Comment:
Why has this output changed?

Reply:
The nulls are the top two elements, whereas previously they were ranked lowest, so that decreases the rank of all other values by 2. That's what I meant by "make nulls ranked highest".

Reply:
Check. Makes sense!